.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
We can probably delete this one.

Reply:
okie doke

Reply:
done in https://github.com/mdn/webaudio-examples/pull/92/commits/053ab323bd89e51dfecaac43e8157dcec784b00a